src/components/10-atoms/fieldset/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@axa-ch/fieldset",
+  "version": "0.0.0-beta.0",
+  "description": "> TODO: description",
+  "author": "Pattern Warriors",
+  "homepage": "https://github.com/axa-ch/patterns-library/tree/master/src/components/10-atoms/fieldset#readme",
```

Comment:
here please use `../tree/develop/..` instead of `../tree/master/..` This is also fixed in the todays PR of scaffolding

Reply:
done